Cargo.toml

```diff
@@ -103,8 +108,17 @@ features = [
   'Window',
   "Document",
   "Element",
+  "ReadableStream",
+  "ReadableStreamDefaultController"
 ]
 
+[dependencies.zstd]
+features = [
+  "wasm",
+  "thin"
+]
+default-features = false
```

Comment:
I've been searching for how to fix this issue for so long! It was the biggest headache in trying to update the repo

Reply:
I brought this out into its own PR in https://github.com/kylebarron/parquet-wasm/pull/279